testing/python/language/test_tilelang_language_reduce.py

```
@@ -307,5 +307,90 @@ def kernel(A: T.Tensor((block_M, 64), T.float32), B: T.Tensor((block_M,), T.floa
         tl.compile(k, out_idx=-1, pass_configs=_COMPILE_FLAGS)
 
 
+# ---------------------------------------------------------------------------
+# nan_propagate tests – packed (vsize=2) path for bf16/fp16
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)